linux/jdk/alpine/build.gradle

```diff
@@ -0,0 +1,143 @@
+plugins {
+	id "java"
+}
+
+ext {
+	junitVersion = "5.9.0"
+	testcontainersVersion = "1.17.3"
+	assertjCoreVersion = "3.23.1"
+}
+
+repositories {
+	mavenCentral()
+}
+
+group "org.adoptium"
+version "1.0.0-SNAPSHOT"
```

Comment:
should that go to 1.0.0?

Reply:
I'd prefer to not add dependency changes in this PR. We should add dependabot to handle the gradle updates